composer.json

```diff
@@ -113,5 +113,7 @@
     "bin": ["bin/rector"],
     "config": {
         "sort-packages": true
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
We shouldn't do this. As we discussed previously, open the `minimum-stability` is a BC.

Reply:
These won't get to tagged version